### PR TITLE
Navbar styling

### PR DESCRIPTION
**Pull request by ManApart**

### CODE REVIEW

File: src/components/header/header.scss [hunk 1]
```diff
@@ -21,33 +21,51 @@ header {
...
+  .highlighted:hover {
+    background-color: black;
+    color: white;
   }
 
   .pinned {
     position: fixed;
     top: 0;
     right: 0;
+    padding:4px 20;
```
Comment (bbrewer-papa-x) [inline src/components/header/header.scss]: This is a nitpick, but `20` of what?

Comment (ManApart) [reply src/components/header/header.scss]: Oh good catch.

File: src/components/header/header.js [hunk 2]
```diff
@@ -58,12 +65,41 @@ export default class extends Component {
...
+
     return (
       <div className={`nav-wrapper ${this.state.collapsed ? 'pinned' : ''}`}>
-        <a className='nav-element' href="https://www.dev-smartos.com/index.php/data">DATA</a>
-        <a className='nav-element' href="https://www.dev-smartos.com/index.php/data-stories">IMPACT</a>
-        <AboutDropdown />
-        <a className='nav-element' href="https://www.dev-smartos.com/index.php/share-your-data">GET INVOLVED</a>
+        <NavDropdown data={aboutData} />
+        <a className='nav-element highlighted' href="https://www.dev-smartos.com/index.php/data">EXPLORE DATA</a>
+        <a className='nav-element' href="https://www.dev-smartos.com/index.php/share-your-data">SHARE DATA</a>
```
Comment (ManApart) [inline src/components/header/header.js]: Need to point to base URL

Comment (smillardHNTB) [reply src/components/header/header.js]: should not have index.php in links either

**Pull request merged**